sidebars.json

```diff
@@ -47,6 +47,11 @@
       "Scripting/BehaviorScriptBasics",
       "Scripting/ControllingAnimations",
       "Scripting/NodeSDK"
+    ],
+    "Enklu Embedded (Preview)": [
```

Comment:
Is this a preview? And if it is, what is the criteria for it leaving preview? Unfortunately `Enklu Embedded (Preview)` is the exact perfect width to make the sidebar dropdown icon look a little weird. Not a big deal, but if it were shorter or longer it'd be fine 😅